### PR TITLE
Vacuum status

### PR DESCRIPTION
**Pull request by schuetzepaul**
This PR introduces a few changes to how the status of vacuum channels are treated.
New features in the background are ...
 * The status of the relay card device can now be queried
 * There's now a `enum SwitchState` that can be used to indicate the status of individual vacuum channels (relays)
 * New status: "switching"

In the GUI this leads to the following changes:
 * In case of an unavailability of the relay card device ... (see issue #195 )
   * there's a warning during startup
   * the vacuum toggle button is greyed out
   * the status indicators turn grey
 * There's a new status "switching" for vacuum channels, which is indicated by an orange colour.

This closes issue #195 .

### CODE REVIEW

File: assembly/assemblyCommon/RelayCardManager.h [hunk 1]
```diff
@@ -18,6 +18,13 @@
 #include <QObject>
 #include <QTimer>
 
+enum class SwitchState {
+  DEVICE_OFF = 0             // Device is off
+  , CHANNEL_OFF = 1          // Channel is off
+  , CHANNEL_ON = 2           // Channel is on
+  , CHANNEL_SWITCHING = 3    // Channekl is currently switching
```
Comment (anurnberg) [inline assembly/assemblyCommon/RelayCardManager.h:25]: Typo in the comment, nothing serious ;)

**Review by anurnberg — APPROVED** [review]
Looks good, as far as I can tell

File: assembly/assemblyCommon/RelayCardManager.h (hunk 1 above)
Comment (schuetzepaul) [reply assembly/assemblyCommon/RelayCardManager.h]: Fixed, thanks :grimacing:

**Comment by schuetzepaul** [comment]
The most recent changes merge #193 and #196 properly, as this invoked a conflict. Also, now the `SWITCHING` state is also represented in the toolbar.

**Pull request merged**